react-native-fbsdklogin/js/FBSDKLoginButton.ios.js

```diff
@@ -29,10 +28,32 @@ var {
   StyleSheet,
 } = React;
 
+import type {
+  FBSDKLoginCallback,
+  FBSDKLoginBehavior,
+  FBSDKDefaultAudience,
+} from './FBSDKLoginManager.ios.js';
+
+type FBSDKLoginButtonProps = {
+  readPermissions: Array<string>;
+  publishPermissions: Array<string>;
+  onLoginFinished: FBSDKLoginCallback;
+  onLogoutFinished: () => void;
+  loginBehavior: FBSDKLoginBehavior,
+  defaultAudience: FBSDKDefaultAudience,
+  tooltipBehavior: $Enum<{
```

Comment:
I don't know. AFAIK, the advantage of `$Enum` in general is that it's easier to use if you already have a type that describes all possible values. In this case, I really don't know.